docs/development/management-domains/README.md

```diff
@@ -17,14 +17,52 @@ There are several reasons why a company might consider the use of Management Dom
   different intervals or velocities. **Note that this only refers to minor changes of APIs, SPIs, configuration etc. All
   instances must still maintain protocol (DSP, DCP,...) compatibility!**
 
-For the purposes of Tractus-X, the usage of
-deployment [type 2b](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/management-domains/management-domains.md#type-2b-edc-catalog-server-and-controldata-plane-runtimes)
-or [type 2c](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/management-domains/management-domains.md#type-2c-catalog-servercontrol-plane-with-data-plane-runtime)
-is assumed:
 
-![type 2b](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/management-domains/distributed.type2.b.svg)
+### Deployment Topologies
```

Comment:
I don't think there is much value in copy-pasting documentation content that is already present in the upstream repo, especially picture material which might be copyrighted. 

Instead, I would document how to set up a catalog server, i.e. which assets must be added there, how the resulting federated catalog looks like (flattened vs hierarchical), etc.